packages/api/src/command/medical/cq-directory/update-organization.ts

```diff
@@ -0,0 +1,27 @@
+import { validateVersionForUpdate } from "../../../models/_default";
+import { CQDirectoryModel } from "../../../models/medical/cq-directory";
+import { CQDirectoryOrg } from "./create-cq-organization";
+
+export const updateCQDirectoryOrganization = async ({
+  existingOrg,
+  newData,
+}: {
+  existingOrg: CQDirectoryModel;
+  newData: CQDirectoryOrg;
+}): Promise<CQDirectoryModel> => {
```

Comment:
Ideally we would:
- encapsulate the logic of this function, by not requiring the client to send a Model that we'll use to update;
- use domain/interfaces - not Models, in and outwards